Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp

```diff
@@ -199,6 +200,13 @@ void ScrollingStateScrollingNode::setSynchronousScrollingReasons(OptionSet<Synch
 }
 #endif
 
+
+void ScrollingStateScrollingNode::setKeyboardScrollData(const RequestedKeyboardScrollData& scrollData)
```

Comment:
This could be use `RequestedKeyboardScrollData&&` with `WTFMove` at the call site and `WTFMove` into the member variable.